iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AbstractLineConnectionModification.java

```diff
@@ -98,18 +98,26 @@ public String getLine2Name() {
         return line2Name;
     }
 
-    private static double checkPositionPercent(double positionPercent) {
+    private static boolean checkPositionPercent(double positionPercent, boolean throwException, Reporter reporter, Logger logger) {
```

Comment:
Instead of passing the logger, you could use a logger defined in `AbstractLineConnectionModification`

Reply:
Ok, that was the case already, and that's not the topic here.